dev-haskell/deferred-folds/deferred-folds-0.9.18.7-r1.ebuild

```diff
@@ -17,7 +17,7 @@ KEYWORDS="~amd64"
 
 RDEPEND=">=dev-haskell/foldl-1:=[profile?] <dev-haskell/foldl-2:=[profile?]
 	>=dev-haskell/hashable-1:=[profile?] <dev-haskell/hashable-2:=[profile?]
-	>=dev-haskell/primitive-0.6.4:=[profile?] <dev-haskell/primitive-0.10:=[profile?]
+	>=dev-haskell/primitive-0.6.4:=[profile?]
```

Comment:
I see the upper bound removed, but no revision bump from Hackage or `CABAL_CHDEPS`.